public/javascripts/SVLabel/build/SVLabel.js

```diff
@@ -11921,27 +11964,29 @@ function ModalMissionComplete ($, d3, L) {
      * @private
      */
     function _updateTheMissionCompleteMessage() {
+        var weirdLineBreakMessages = [
```

Comment:
Messages I took out because they were too long or had weird line breaks in the new width. I kept them so we still have them.